src/NzbDrone.Core/Datastore/Migration/203_add_release_hash.cs

```diff
@@ -0,0 +1,14 @@
+using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(203)]
```

Comment:
I wasn't sure how DB migrations are kept in sync across forks, so I just incremented to the next ID.

Reply:
203 is already used in https://github.com/Sonarr/Sonarr/pull/6567 will need to be bumped to 204